taccsite_cms/templates/djangocms_blog/post_list.html

```diff
@@ -1,41 +1,35 @@
-{% extends "djangocms_blog/base.html" %}
-{% load i18n easy_thumbnails_tags %}{% spaceless %}
-
-{% block canonical_url %}<link rel="canonical" href="{{ view.get_view_url }}"/>{% endblock canonical_url %}
+{# https://github.com/nephila/djangocms-blog/blob/1.1.1/djangocms_blog/templates/djangocms_blog/post_list.html #}
```

Comment:
**Note.** This is a diff of (1) the markup at this URL (which I copied-but-did-not-use years ago) and (2) the markup changes I must make now to support the design.